static/data/grunttype.json

```diff
@@ -33,6 +33,7 @@
   },
   "6": {
     "type": "Bug",
+    "name": "Bug",
```

Comment:
Dont think we need this right? We can just use type

Reply:
Oh that's testing garbage lol